common/prettify.pm

```diff
@@ -620,24 +623,36 @@ sub Footer ()
 {
     my $self = shift;
     my $out = $self->{FH};
+    my $host = hostname;
+    if (!defined $host) {
+        $host = 'localhost';
+    }
+    my $git_hash = main::GetVariable ('TRIGGERING_COMMIT');
```

Comment:
What when multiple repositories are polled for a change?

Reply:
replaced with array